eclair-core/src/main/scala/fr/acinq/eclair/io/Monitoring.scala

```diff
@@ -21,8 +21,10 @@ import kamon.Kamon
 object Monitoring {
 
   object Metrics {
-    val PeerConnections = Kamon.counter("peers.connecting.count")
-    val ConnectedPeers = Kamon.gauge("peers.connected").withoutTags()
+    val PeerConnectionConnecting = Kamon.counter("peerconnection.connecting")
```

Comment:
I wouldn't not explicitly mention `peerconnection` instead of `peer` here, it's quite an implementation detail that people looking at the monitoring graphs don't even need to know about. This is really a peer connecting, so `peers.connecting.count` seemed better IMHO.

Reply:
Yes but this one provides information about the actual connection state (authenticating/authenticated/...)  and is "richer" than the peer metrics which is just off/on.

Reply:
Yes but it's mostly about the naming, I would have just named this `peers.connecting.count`. The ops looking at the graph doesn't necessarily need to know that `peer` and `peerconnection` are different things. But it's probably nit.

Reply:
@t-bast The problem with your suggestion is that we also need to track the state "connected" in the `PeerConnection` as I explained https://github.com/ACINQ/eclair/pull/1357#discussion_r398743209.

Reply:
That's only an argument for keeping the peerconnection gauge, not really for the naming of this metric. But anyway this is nit, not worth wasting time bikeshedding.